.setup/vagrant.sh

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/env bash
```

Comment:
Is this script used for the production (non vagrant) installation too?  Would it be better to name the file something without the word "vagrant"?  Maybe install_system.sh?  (or capitalized?)